templates/embeds/card.html.twig

```diff
@@ -59,9 +60,11 @@
 
     {# Body #}
     {% block box_body_before %}{% endblock %}
-    <div class="{% block box_type_class %}card-body {% endblock %}{% block box_body_class %}{% endblock %} {% if _fullsize %}p-0{% endif %} {% if _collapsible %} {{ _collapsible_class ~ (_collapsed ? ' collapse' : ' show') }} {% endif %}">
-        {{ body|raw }}
-    </div>
+    {% if _hasBody %}
```

Comment:
stop, one more thing: this is a BC break.
Now the body will be removed if it is empty as well. Why did we not use the `_hasBody` variable before? 
Did we not add that check before on purpose? 

I just checked my app and it will likely not hit me, but I could imagine, that people might have used the embed without defined body and then use javascript to write into it, e.g. for dynamic modal content?
You think that might become a problem?

Reply:
That was in my first commit and PR version
> Can be BC break, if some JS is made to fill that card-body 🤔

ATE I've removed this mention because `use_body` is set to `true` by default.
So it is the same behavior as today. (no BC break then)

In MY case I have to set use_body to `false` to not print it.

In the V2 card we can remove this 👍

Reply:
1. What is [ATE](https://www.abbreviationfinder.org/acronyms/ate.html)?
2. What a long story. Sorry! I didn't get any of it in the first place I guess. Probably I am working too much on my own 😁 Actually using force_body from your initial commit sounds more reasonable to me for the long term. If the body block is not defined, than the container will not be rendered, unless you force it to.

I am fine with both solutions, actually that would be a minor BC, which I would be okay with. 

You are the maintainer as well, so what do you think is better? 
use_body = true || force_body = false